getting-started/kubernetes/customizations/imageset.md

```diff
@@ -0,0 +1,199 @@
+   ```
+       imageSet: calico-{{site.data.versions.first.title}}
+   ```
+
+#### Update the operator deployment with a digest
+
+Before applying `tigera-operator.yaml`, modify the operator deployment to use the operator image digest.
+
+Use commands like the following to get the image digest (adjust the image in the commands if you are using a different operator image):
+```bash
+docker pull {{ operator.registry }}/{{ operator.image }}:{{ operator.version }}
+docker inspect {{ operator.registry }}/{{ operator.image }}:{{ operator.version }} -f {% raw %}'{{range .RepoDigests}}{{printf "%s\n" .}}{{end}}'{% endraw %}
+```
+
+If there are multiple digests returned select the one matching the repository you are using.
+
+Update the tigera-operator deployment:
+```bash
+sed -ie "s|\(image: .*/operator\):.*|?\1@<put-digest-here>|" tigera-operator.yaml
+```
+
+#### Create an ImageSet manifest
+
+Create an [ImageSet]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.ImageSet) manifest file named `imageset.yaml` like the following:
+
+```yaml
+apiVersion: operator.tigera.io/v1
+kind: ImageSet
+metadata:
+  name: calico-{{site.data.versions.first.title}}
+spec:
+  images:
+  - image: "calico/cni"
+    digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+  - image: "calico/kube-controllers"
+    digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+  - image: "calico/node"
+    digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+  - image: "calico/typha"
+    digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+  - image: "calico/pod2daemon-flexvol"
+    digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+  - image: "tigera/operator"
+    digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+  - image: "tigera/key-cert-provisioner"
+    digest: "sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
+```
+
+1. Copy the above example into a file called `imageset.yaml` and edit that file in the steps below.
+1. Set the name for your ImageSet to `calico-<version>` (Example: `calico-{{site.data.versions.first.title}}`).
+   The version can be obtained by running:
+   ```
+   docker run {{ operator.registry }}/{{ operator.image }}:{{ operator.version }} --version
+   ```
+1. Add the correct digest for each image. If you are using a private registry ensure you pull the image from the private registry and use the digest associated with the private registry.
+     1. If using the default images, get a list of them by running:
+        ```
+        docker run {{ operator.registry }}/{{ operator.image }}:{{ operator.version }} --print-images=list
+        ```
+        >**Note**: If you are not using the default image registries or paths you will need to come up with the list of images and cannot rely on the above command.
```

Comment:
If you are not using the default image registries or paths, you need to come up with the list of images, and above command will not apply.??